### DISCUSSION TITLE
in session configuration, can I split window previous one?

### DISCUSSION THREAD

**Discussion by nolleh** [kovidgoyal/kitty#6062] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I want to split window  like this when session startup.

```bash
|----------|
|    |     |
|----|     |
|    |     |
|----|-----|
```

is there means to do that? (it is possible that just I can't find out the way. )

**Describe the solution you'd like**
if we have in a focusing command in session.conf,
looks like it will work as expected.
for example

```conf
new tab
launch --title=lefttop zsh
launch --title=right zsh
focus lefttop ### <------this!
launch --title=leftbottom --location=hsplit zsh
```

thx in advance!

**Comment by kovidgoyal** [comment, marked as answer]
Set the layout to tall mirrored and do three launches

**Reply by nolleh** [reply]
oh, thanx. 

```
new tab
enabled_layouts tall:bias=68;mirrored=true, stack
launch --title=right zsh
launch --title=lefttop zsh
lauhcn --title=leftbootom --location=hsplit zsh
```

this configuration do excactly what I want !
